src/rules/softdot/gamma.jl

```diff
@@ -0,0 +1,14 @@
+# Belief propagation: does not exist for softdot
```

Comment:
Good comment, although then be consistent and say which kind of variational rule is available, i.e., mean-field variational message